lib/ansible/modules/git.py

```diff
@@ -1238,6 +1238,10 @@ def main():
 
     result = dict(changed=False, warnings=list())
 
+    valid_urls = ("https://", "http://", "file://", "/", "git", "ssh://", "~",)
```

Comment:
Git can also clone a relative path, is that possible with the module?

Reply:
No, I think validating the URL repo is not working as intended. We can bikeshed this idea.